app/editor/src/features/admin/users/UserList.tsx

```diff
@@ -16,6 +16,8 @@ export const UserList: React.FC = () => {
   const navigate = useNavigate();
   const [{ users, userFilter }, { findUsers, storeFilter }] = useUsers();
   const [{ requests }] = useApp();
+
+  const [filter, setFilter] = React.useState(userFilter);
```

Comment:
The page will manage it's own state to stop infinite loops.  I would like to find out why it loops at some point.